<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PolygonDialog</class>
 <widget class="QDialog" name="PolygonDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="sides">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>330</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QLabel" name="configPoligono1">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>330</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Lados</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="size">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>300</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>50</string>
   </property>
  </widget>
  <widget class="QLabel" name="configPoligono2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>300</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tamaño</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>160</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Dist. X</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="translateX">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>160</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>20</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>160</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Dist. Y</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="translateY">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>160</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="translateButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>190</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Trasladar</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ángulo</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rotationAngle">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>20</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rotateButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Rotar</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>160</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>X:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="scaleX">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>160</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>160</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="scaleY">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>160</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scaleButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>190</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Zoom</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="reflectionXButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Reflexión X</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="reflectionYButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Reflexión Y</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Polígono</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Arco</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cono</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cubo</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Prisma Rectangular</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Prisma Triangular</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="arcSize">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>300</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>200</string>
   </property>
  </widget>
  <widget class="QLabel" name="configArco2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>300</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tamaño</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="radius">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>330</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>90</string>
   </property>
  </widget>
  <widget class="QLabel" name="configArco1">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>330</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Radio</string>
   </property>
  </widget>
  <widget class="QLabel" name="confingPoligono3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>270</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Config. Polígono</string>
   </property>
  </widget>
  <widget class="QLabel" name="configArco3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Config. Arco</string>
   </property>
  </widget>
  <widget class="QLabel" name="configPrismas">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tamaño</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="prismSize">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>300</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>50</string>
   </property>
  </widget>
  <widget class="QLabel" name="configPrismas1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Config. Prismas</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearTransformsButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>270</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Borrar Transformaciones</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="updatePoly">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>491</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Visualizar</string>
   </property>
  </widget>
  <widget class="QLabel" name="configPrismas1_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Escoge la figura:</string>
   </property>
  </widget>
  <widget class="QLabel" name="configPrismas1_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Transformaciones</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
